### ISSUE TITLE
Relationship/overlap between 2.1's "Pointer Gestures" and 2.2's "Dragging"

### ISSUE THREAD

**Issue by patrickhlauke** [w3c/wcag#1398]
2.5.1 Pointer Gestures talks about (but does not normatively define) gestures including "swiping, sliders and carousels dependent on the direction of interaction, and other gestures which trace a prescribed path such as the drawing a specific shape".

particularly the "carousel" mention there sounds like it overlaps with "Dragging", which includes examples like a radial widget and a map, (which are not completely/clearly "elements" that are dragged per se, same way that scrubbing through a carousel is not necessarily a "gesture" perhaps)?

confusing, the understanding for 2.5.7 Dragging includes mention of "gesture": "When an interface implements functionality that uses dragging motions, some users can tap or click, but not accurately maintain contact whilst performing a gesture."

so in general...some more explanation of why these are different, where their overlap is, and what makes one be a level A and the other a AA, would be necessary here.

**Comment by becka11y** [comment]
I completely agree with @patrickhlauke - there needs to be a better distinction between pointer gestures and dragging. At this point, I can't figure out how I would describe this SC to a non-developer.  The updates suggested by #1316 do help but the SC would benefit from more examples. Especially since, two of the current examples are very similar to ones in the Pointer Gestures understanding document.

**Comment by stringyland** [comment]
I'm also interested to know why Dragging is at AA although Pointer Gestures is A. If it is intended to close a loophole, shouldn't it be at the same level?

**Comment by detlevhfischer** [comment]
The genesis of Pointer Gestures is that some in the WG felt the need to differentiate gestures like swipe from dragging. I would have preferred to have one SC for all actions (whether swipe, drag or multitouch) that involve complex pointer movements across the surface, but we ended up making initial directionality the decisive criterion for a 'gesture'. Which leaves us with a situation where in some usage contexts like touch on mobile OSs, something (like panning a carousel) can be considered a gesture because going straight up fails to engage the carousel but pans the page instead, while in other usage contexts, it would be considered a dragging motion (not a gesture) because once the pointer has come down on the target, it will follow the pointer (possibly constrained to one axis) whatever way the pointer moves, i.e., no directionality.

Having to explain that in an Understanding text is complex and awkward at any rate, and I wish we'd never gotten ourselves into this corner, but here we are. What we can certainly do is (A) remove mention of 'gesture' in the Dragging understanding text, and (B) refer to the same lengthy and hair-splitting disambiguation of 'what is a gesture' (and by implication, what is not) that can be found in the understanding text of "Pointer Gestures":
WCAG 3.0 will hopefully then have just one Guideline (or whatever it'S going to be called) for both Pointer Gestures and Dragging.

**Comment by detlevhfischer** [comment]
Have now implemented the changes suggested in https://github.com/w3c/wcag/pull/1423

**Comment by AndrewArch** [comment]
Looking forward to have a combined Guideline in WCAG 3.0 :)

**Comment by alastc** [comment]
#1423 has been merged in, closing this issue. Please re-open if something has not been addressed.
